### DISCUSSION TITLE
Code conventions

### DISCUSSION THREAD

**Discussion by Alteras1** [RpNation/bbcode#16] [General]
As @MShultz  pointed out, our previous styling and class names were severely inconsistent. We should probably unify on a common code convention as we're coding everything again.

https://github.com/RpNation/bbcode/wiki/Code-Conventions

**Comment by Alteras1** [comment]
## Issues for styling:
- prefix class name
- inline styles for small micro styles??? i.e. ([left] and [right] just being `text-align: left` and `text-align: right`)

**Reply by MShultz** [reply]
For now, I've just gone ahead with bb-* as the styling convention. You can see those for alignment and highlight tags so far. 

For inline vs style sheets, for now I just followed the convention we had in the old plugin (all styles were in one sheet and we tried to avoid inlining). I think it helps future devs know where to look first for style changes/extensions without having to worry about unseen impacts. That said, as I've looked through the old plugin again, we didn't always follow that convention (pindent, for example). Both implementations are just as easy, so inline styles are doable if we choose to go that route to reduce the number of files in the solution.
